src/styles.scss

```diff
@@ -2633,10 +2633,10 @@ th.mat-sort-header-sorted {
   color: $primary-color;
 }
 
-// .mat-table {
-//   overflow: auto;
-//   height: 550px;
-// }
+.mat-table {
+  overflow: auto;
+  height: 550px;
```

Comment:
Why is this hardcoding of height

Reply:
To fix the height of the table Even few records or no records